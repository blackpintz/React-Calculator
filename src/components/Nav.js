import React from 'react';
import styled from 'styled-components';
import { Link } from 'react-router-dom';

export default function Navbar() {
  const Nav = styled.nav`
    display: flex;
    width: 90%;
    margin: 0 20px;
    box-sizing: border-box;
    & h2 {
        margin: 0;
        flex-grow: 1;
    }
    & a {
        padding: 9px 5px 0;
    }
    & span {
        padding-top: 9px;
    }
  `;
  return (
    <Nav>
      <h2>Math Magicians</h2>
      <Link to="/">Home</Link>
      <span>|</span>
      <Link to="/calculator">Calculator</Link>
      <span>|</span>
      <Link to="/quote">Quote</Link>
    </Nav>
  );
}
