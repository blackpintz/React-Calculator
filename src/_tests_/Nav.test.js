import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from '../components/Nav';

test('first snapshot test', () => {
  const component = renderer.create(
    <BrowserRouter><Nav /></BrowserRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
