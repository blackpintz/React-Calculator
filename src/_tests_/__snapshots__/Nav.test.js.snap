// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first snapshot test 1`] = `
<nav
  className="sc-bdfBwQ dgOyCB"
>
  <h2>
    Math Magicians
  </h2>
  <a
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
  <span>
    |
  </span>
  <a
    href="/calculator"
    onClick={[Function]}
  >
    Calculator
  </a>
  <span>
    |
  </span>
  <a
    href="/quote"
    onClick={[Function]}
  >
    Quote
  </a>
</nav>
`;
